scrapeJEmails-NoTechOrBlog.py

```diff
@@ -0,0 +1,129 @@
+import os 
+import re
+from bs4 import BeautifulSoup
+import sys
+
+
+def scrapeEmailsFromSinglePage(linkToScrape): 
+
```

Comment:
i'd love to see this method refactored into smaller methods. with particular care to avoid heavy levels of nesting.  see lines 51 - 94.